core/platform-common/src/main/resources/externalresource.properties

```diff
@@ -116,4 +116,6 @@ user_read_api_v2_non_mandatory_fields= profileDetails.verifiedKarmayogi,profileD
   profileDetails.professionalDetails[0].designation,profileDetails.professionalDetails[0].location,\
   profileDetails.professionalDetails[0].doj,profileDetails.professionalDetails[0].description,profileDetails.employmentDetails.departmentName
 kafka_user_first_login_event_topic= dev.user.first.login
+otpValidationSecretKey=secretKey
+otpExpirationTimeToken=300000
```

Comment:
change this value to 5 mins

Reply:
its for 5mins only karthik i check on google for value.